byway/src/main/java/com/google/sps/data/Trip.java

```diff
@@ -74,6 +74,15 @@ public Trip(
     return Collections.unmodifiableList(this.route);
   }
 
+  /**
+   * Sets the route of the trip with a list of stops and destinations as plain text.
+   *
+   * @param route list of user-selected stops and destinations
+   */
+  public void setRoute(Collection<String> route) {
+    this.route = new ArrayList<>(route);
+  }
```

Comment:
Same as other PR, keep it final, use `clear` + `addAll`.

Reply:
Done.